### PR TITLE
Update types/pubnub

### PR DESCRIPTION
**Pull request by mohitpubnub**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:[PubNub configuration](https://www.pubnub.com/docs/sdks/javascript/api-reference/configuration#cryptomodule-configuration)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/pubnub/pubnub-tests.ts
1. types/pubnub/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d4971483b95395f277c6f92c72ed7af4fc6dfac9
</p>

**Review by Salet — APPROVED** [review]
Looks good, thanks for the changes

**Comment by mohitpubnub** [comment]
Ready to merge

File: types/pubnub/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "webworker"
```
Comment (andrewbranch) [inline types/pubnub/tsconfig.json:6]: What was needed from `webworker`, and is it available in `dom`? The bot is complaining because of this.

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/pubnub/tsconfig.json (hunk 1 above)
Comment (mohitpubnub) [reply types/pubnub/tsconfig.json]: Yes, `dom` should be enough for this.
Thanks!

**Review by Salet — APPROVED** [review]

**Comment by mohitpubnub** [comment]
Ready to merge

**Pull request merged**